docs/source/resources/releases_maintenance.rst

```diff
@@ -6,39 +6,95 @@
 at the same time as the dependency, 1 year later.
 
+Life Cycle Phase
+================
+
+To encourage the adoption of new features while keeping the high standard of stability inherent,
+support is divided into life cycle phases; **full support** which covers the first year
+and **maintenance support** which covers the second year.
+
++--------------------------+------------------------------------+---------------------------+
+| Life Cycle Phase         | Full Support                       | Maintenance Support       |
++==========================+====================================+===========================+
+| Critical security fixes  | Yes                                | Yes                       |
++--------------------------+------------------------------------+---------------------------+
+| Bug fixes by severity    | Critical and high severity issues  | Critical severity issues  |
++--------------------------+------------------------------------+---------------------------+
+
+Severities
+==========
+
+Severity 1 (Critical):
+A problem that severely impacts your use of the software in a production environment (such as loss
+of production data or in which your production systems are not functioning). The situation halts
+your business operations and no procedural workaround exists.
+
+Severity 2 (high):
+A problem where the software is functioning but your use in a production environment is severely
+reduced. The situation is causing a high impact to portions of your business operations and no
+procedural workaround exists.
+
+Severity 3 (medium):
+A problem that involves partial, non-critical loss of use of the software in a production environment
+or development environment and your business continues to function, including by using a procedural
+workaround.
+
+Severity 4 (low):
+A general usage question, reporting of a documentation error, or recommendation for a future product
+enhancement or modification.
+
+Severities 3 and 4 are generally addressed in subsequent releases to ensure a high standard of stability
+remains available for production environments.
+
+Support Matrix
+==============
+
 These are the component versions available when the collection was made generally available (GA). The underlying
 component version is likely to change as it reaches EOL, thus components must be a version that is
 currently supported.
 
 For example, if a collection releases with a minimum version of ``ansible-core`` 2.14.0 (Ansible 7.0) and later this
 enters into EOL, then a newer supported version of ``ansible-core`` (Ansible) must be selected. When choosing a newer
 ``ansible-core`` (Ansible) version, review the `ansible-core support matrix`_ to select the appropriate dependencies.
-This is important to note, different releases of ``ansible-core`` can require newer controller and managed node
+This is important to note, different releases of ``ansible-core`` can require newer control node and managed node
```

Comment:
i'm curious as to the reasoning behind this change from controller to control node, personally i find it easier to use 'controller' and 'target node' to keep the two straight in my head (as opposed to control node and target or managed node)

Reply:
Not sure why @ddimatos changed this, reviewing the Ansible docs I see it mentioned as **controller** but when talking about architecture is mentioned as **control-node** [here](https://www.ansible.com/how-ansible-works/). and **managed node** seems to be the preferred word.

Reply:
controller implies human (one who controls).  control node implies a machine.